Cargo.toml

```
@@ -56,7 +57,7 @@ matchit = "0.7.3"
 p256 = { version = "0.13.2", features = ["ecdsa-core", "ecdsa", "pem"] }
 paste = "1.0.14"
 pin-project = "1.1.5"
-prio = "0.16.4"
+prio = "0.16.3"
```

Consider pinning the version, and perhaps the getrandom version.
```suggestion
prio = "=0.16.3"
```